### PR TITLE
Dima

### PR DESCRIPTION
**Pull request by DmytroKaliuzhnyi**
add
@dimaserbenyuk

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1 +1,5 @@
-hi
\ No newline at end of file
+hi
+Hello
+test2
```
Comment (dimaserbenyuk) [inline test.txt:3]: good

File: test.txt [hunk 2]
```diff
@@ -1 +1,5 @@
-hi
\ No newline at end of file
+hi
```
Comment (dimaserbenyuk) [inline test.txt:1]: `add some`

**Review by dimaserbenyuk — APPROVED** [review]
good

**Comment by DmytroKaliuzhnyi** [comment]
tets2

**Pull request merged**